portals/new_admin/portals/client/src/App.css

```diff
@@ -0,0 +1,469 @@
+    border-radius: 100px;
+    padding: 10px 40px 10px 40px;
+  }
+
+  nav .links a {
+    font-family: "Times New Roman", Times, serif;
+    font-size: 5rem;
+    color: white;
+    font-weight: 300;
+  }
+  nav .icon-burger {
+    z-index: 9;
+    display: block;
+    caret-color: transparent;
+  }
+  nav :checked ~ .links {
+    left: 0;
+  }
+  nav .icon-burger .line {
+    transition: 0.3s cubic-bezier(0.97, 0.02, 0.44, 0.99);
+  }
+  nav :checked ~ .icon-burger .line:nth-child(1) {
+    transform: translateY(9px) rotate(50deg);
+  }
+  nav :checked ~ .icon-burger .line:nth-child(2) {
+    transform: translateX(10px);
+    opacity: 0;
+  }
+  nav :checked ~ .icon-burger .line:nth-child(3) {
+    transform: translateY(-9px) rotate(-50deg);
+  }
+}
+
+/* ----------header---------------- */
+
+.header {
+  display: flex;
+  flex-direction: column;
+}
+
+.hero {
+  width: 100%;
+  height: 100vh;
+  position: relative;
+  overflow: hidden;
+}
+
+.hero .circles {
+  width: 100px;
+  height: 100px;
+  border-radius: 100%;
+  position: absolute;
+  border: 3px solid rgb(255, 255, 255);
+  background: linear-gradient(148.92deg, #58bbf2 13.69%, #00278c 88.85%);
+  box-shadow: 0px 0px 135px rgba(0, 0, 0, 0.35);
+  animation: circles 1s cubic-bezier(0.9, 0.02, 0.19, 1);
+}
+
+@keyframes circles {
+  from {
+    transform: translateY(120%);
+  }
+  to {
+    transform: translateY(0px);
+  }
+}
+
+.hero .circles:nth-child(1) {
+  top: 50%;
+  left: -5%;
+  width: 50rem;
+  height: 50rem;
+  z-index: 2;
+  animation: circles 0.8s cubic-bezier(0.9, 0.02, 0.19, 1);
+}
+.hero .circles:nth-child(2) {
+  top: 25rem;
+  left: 20rem;
+  width: 50rem;
+  height: 50rem;
+  z-index: 3;
+  animation: circles 1s cubic-bezier(0.9, 0.02, 0.19, 1);
+}
+.hero .circles:nth-child(3) {
+  top: 40%;
+  right: -5%;
+  width: 80rem;
+  height: 80rem;
+  z-index: 4;
+  animation: circles 1.2s cubic-bezier(0.9, 0.02, 0.19, 1);
+}
+.hero .circles:nth-child(4) {
+  top: -20%;
+  left: -10%;
+  width: 60rem;
+  height: 60rem;
+  z-index: 0;
+  animation: circles 1.4s cubic-bezier(0.9, 0.02, 0.19, 1);
+}
+.hero .circles:nth-child(5) {
+  top: -30%;
+  left: 18%;
+  width: 90rem;
+  height: 90rem;
+  z-index: 0;
+  animation: circles 1.5s cubic-bezier(0.9, 0.02, 0.19, 1);
+}
+.hero .circles:nth-child(6) {
+  top: -10%;
+  left: 60%;
+  width: 60rem;
+  height: 60rem;
+  z-index: 0;
+  animation: circles 1.8s cubic-bezier(0.9, 0.02, 0.19, 1);
+}
+
+.header h1 {
+  position: absolute;
+  z-index: 16;
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+    Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  text-transform: capitalize;
+  top: 30rem;
+  right: 20rem;
+  font-size: 3rem;
+  width: 35vw;
+  text-align: end;
+  color: white;
+  font-weight: 500;
+  animation: h1 1.2s cubic-bezier(0.9, 0.02, 0.19, 1);
+}
+
+@keyframes h1 {
+  from {
+    transform: translateY(180%);
+  }
+  to {
+    transform: translateY(0);
+  }
+}
```

Comment:
Shall we add an EOF new line here as well?

Reply:
Currently, there is no such file.